docker/Dockerfile-build

```diff
@@ -37,7 +37,10 @@ RUN sudo apt-get update -qq \
     && sudo apt-get clean
 
 # This should automatically install the version specified in rust-toolchain.toml
+ADD rust-toolchain.toml rust-toolchain.toml
```

Comment:
We need to send the file in to the docker image explicitly, which means a bit of a change to how the image gets built (see updated README).